tests/tasks/setup/eks/awscli-pod-identity-association.yaml

```diff
@@ -0,0 +1,87 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: awscli-eks-pia-create
+  namespace: scalability
+spec:
+  description: |
+    Create an EKS Pod Identity Association for a given cluster.
+    This Task can be used to create an EKS Pod Identity Association for namespace default and service account default.
+  params:
+  - name: cluster-name
+    description: The name of the EKS cluster you want to create an EKS Pod Identity Association for.
+  - name: region
+    default: "us-west-2"
+    description: The region where the cluster is in.
+  - name: endpoint
+    default: ""
+  - name: namespace-prefix
+    default: "default"
+  - name: namespaces
```

Comment:
How about namespace-count ? or something that tells its the count of namespaces

Reply:
changed